bundle/edu.gemini.spModel.core/src/main/scala/edu/gemini/spModel/core/package.scala

```diff
@@ -8,4 +11,30 @@ package object core {
   type Dec = Declination
   val  Dec = Declination
 
-}
\ No newline at end of file
+  type Ephemeris = Long ==>> Coordinates
```

Comment:
I had not considered that, but I guess it makes sense now that we're using Java 8. I'll see how it looks.